src/methods/clipping/clipping_processor.jl

```diff
@@ -0,0 +1,295 @@
+# # This file contains the shared helper functions for the other polygon clipping
+# # functionalities.
+
+"""
+    _poly_in_poly(::GI.PolygonTrait, poly_a, ::GI.PolygonTrait, poly_b):: Bool 
+
+    This function is only called when we know the two polygons do not intersect. This
+    returns true if poly_a is completely contained in poly_b and false otherwise.
+"""
+# TODO: tempory _poly_in_poly, only works if entirely contained
+# Checks if polya in poly b
+function _poly_in_poly(poly_a, poly_b)
+    p_a = to_edges(poly_a)[1][1]
+    return point_in_polygon(p_a, poly_b)
+end
+
+"""
+    _my_get_pts(::GI.PolygonTrait, poly_a)::Vector{Tuple{Float64}}
+
+    The GeometryOps to_points was not working so I wrote this one. It just
+    used the 'to_edges' function from GeometryOps to derive to_points. 
+"""
+# TODO: temporary alternative for to_points, repeats first and last point
+function _my_get_pts(poly_a)
```

Comment:
This does something a little different from `_to_points`, even if `_to_points` worked since it also adds the repeated last point. Let's try to fix _to_points and add an optional keyword parameter `repeat_last` that checks if the last point is repeated and adds the point if not.

For `_to_points`, did you switch both instances of `x` to `xs`?

Reply:
I do remember switching both instance of `x` to `xs`, but I have not looked at it in a while. I will try to fix this. So `repeat_last` would be a boolean function argument that specifies whether the user wants the lastpoint repeated or not?

Reply:
Yep! And maybe default would be false? I think this would be helpful for the flexibility. So it it is true, just check if the first and last points are equal and if they aren't already then add the repeated point. If it won't work, just ping me and I will take a look as well